### PR TITLE
[CONTENT] Remove start codes

### LINKED ISSUE #4906: [CONTENT] Remove start_codes, load code in slides differently
### Discussed in https://github.com/hedyorg/hedy/discussions/4837

<div type='discussions-op-text'>

<sup>Originally posted by **Felienne** December  1, 2023</sup>
All adventures currently have a field start_code, see:

https://github.com/hedyorg/hedy/blob/5c49bdce3e3007948fe10d145c00990838c2f574/content/adventures/en.yaml#L30

This is a bit of legacy content, as I have said before, I only realised after a long while that we are building a system for teaching programming, not a system for kids to learn or explore programming alone. These start codes are very old, when we (I) thought that every level would just have a start code and kids would append and explore alone. But our preferred workflow for kids now is to read the exercise, take the startcode from the exercise, and change that!

So, there are a number of reasons to get rid of these start codes now:

* They are not useful anymore; the workflow is based on exercises (see above)
* They are adding a lot of translation efforts! Since we have to translate all of them separately. `'# place your code here'` occurs 73 times in en.yaml and has to be translated each time!

So, let's remove them from the content files!</div>

### PR DESCRIPTION
**Pull request by Felienne**
Fixes #4906 by removing all start_codes and associated test code and validation.

**How to test**
Verify that everything works without using start_codes

### CODE REVIEW

**Comment by Felienne** [comment]
Dropping the current Weblate stats from the readme here because I want to see the impact on the percentages after merge :) 

image

**Comment by Felienne** [comment]
Hi @SabinaChita!

This is a small change in content, but it will have quite an impact on teaching, I suspect! Here are some ideas to place in the newsletter, feel free to adapt where needed, the headers are more for you that to be placed in the newsletter like this! The screenshots are also for you, I can imagine we will make other ones for the newxletter.

**How it used to work**
All levels used to have a start_code, if you go to a new adventure, code would be shown in the editor:

image

**How it works now**
There are no more start_codes, so if you go to a new tab now, where you have never created something, the editor box will be empty:

image

(will finish the rest later!)
**Why**

**Potential impact on teaching**

File: tests/cypress/e2e/hedy_page/run_code_button.cy.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ describe('Is able to run code', () => {
     it('Passes', () => {
       goToHedyPage();
 
+      cy.get('#editor').type("print Hallo!'\n");
```
Comment (rix0rrr) [inline tests/cypress/e2e/hedy_page/run_code_button.cy.js:7]: I've seen this not work well in the past. In one of my PRs I changed this to something that seems to work better:

```suggestion
      cy.get('.cm-content').click();
      cy.focused().type("print Hallo!'\n");
```

File: tests/cypress/e2e/hedy_page/keyword_language.cy.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
-import {goToHedyPage} from "../tools/navigation/nav";
+import {goToHedyPageWithEnKeywords} from "../tools/navigation/nav";
 
 describe('when the user changes their language to Arabic', () => {
   beforeEach(() => {
-    goToHedyPage();
-
+    goToHedyPageWithEnKeywords();
+    cy.get('#editor').type("print Hallo!'\n");
```
Comment (Felienne) [inline tests/cypress/e2e/hedy_page/keyword_language.cy.js:6]:
```suggestion
    cy.get('.cm-content').click();
    cy.get('#editor').type("print Hallo!'\n");
```

File: tests/cypress/e2e/hedy_page/run_code_button.cy.js (hunk 1 above)
Comment (Felienne) [reply tests/cypress/e2e/hedy_page/run_code_button.cy.js]: Ah great idea! I added the same change to the other Cypress test I updated!

**Review by rix0rrr — APPROVED** [review]

**Pull request merged**